### DISCUSSION TITLE
成立FAMLAND社区安全委员会

### DISCUSSION THREAD

**Discussion by otto-001** [famland/website#155] [Polls]
## 成立FAMLAND社区安全委员会：

造成本次黑客攻击事件的核心原因，是社区安全意识不足，智能合约部署及审计流程不严谨不规范，为根本解决此类问题，社区拟通过从社区主要核心玩家中进行招募选举，成立专门的安全委员会，专职负责社区的安全建设和合约部署更新等常规事宜的流程监督和管理，安全委员会组成成员接受全体社区成员的监督。  

社区安全委员会的成员由 **@盐之有理** 具体负责招募，初期组成人员7-9人，于11月5日前组织社区全体玩家投票产生。委员会成立后，将公示安全委员会成员身份的钱包地址，并将其钱包地址加入到相关钱包的多签账户中。
 
## 社区安全委员会的职能： 

+ 1、监管社区各合约账户变动情况，当流动池合约资产发生10%以上的资金变动，第一时间响应熔断机制，安全委员会成员在不变更合约业务规则的前提下，通过多签方式调整合约权限，临时终止/开启合约正常交易的执行；

+ 2、参与社区专用钱包账户的多签出入金管理，包括社区开发基金账户、社区空投及运营账户等。社区核心合约内资金的变动（包括流动池、蓄水池、分发池、稳定池、奖金池、农场共享池等合约），须由社区安全委员会提请资金变动请求提案，由社区玩家投票，符合通过条件后，方可安排执行。

+ 3、加强合约安全监管，规范合约上线部署流程，技术团队每次合约更新的新代码须由安全委员会提交审计公司初审，初审通过后，再进行正式链的部署，部署后的合约代码，需要审计公司再次复核。

## 社区安全委员会的议事规则： 

涉及社区安全的相关提案，安全委员会成员的有效投票数须达到全体成员2/3以上，方可执行。社区技术团队提供相应的技术支持和配合。
***
## 社区安全委员会选举流程：
在本帖下面跟帖报名。 
+ 跟帖报名时间：2022年11月4日20:00结束
+ 投票时间：报名统计结束---2022年11月5日20：00结束
+ 跟帖内容可以简单自我介绍，
+ 登记用于多签的EOS账号。

**Comment by lijifacai** [comment]
报名者：独立玩家。自我介绍：FAMLAND的忠实玩家，从FAMLAND众筹跟随该项目，为社区建言献策，能够响应社区需要。申请多签账户：zhouliaww1.e

**Comment by famland16816** [comment]
社区昵称：共享主义先进代表

一个坚定的共享主义信仰者。

从刚开始的金榕树，到福德树，再到八宝树，再到现在的famland，八年来一直紧紧追随，朝着梦想的方向，努力前行！

创始人Topal老师提出的三部曲：

八宝树，把人的时间变为数字财富。利用这种激励机制，构建的是人类社会生态的原始萌芽。追求时间价值的最大化，让人类开始信。

福德树，把人的知识变为数字财富。利用这种激励机制，构建的是人类社会生态的主体。追求思想传播量的最大化，让人类开始行。

金榕树，把人的关系变为数字财富。利用这种激励机制，构建的是人类社会生态的未来。追求自我外延的最大化，让人类不断升华。

famland核心是共享，增量新财富。
把人从赚钱的机器中解放出来。
有更多的时间来实现这三部曲！

期待更多的伙伴加入我们，一起共创，共建，共有，共享美好未来！

多签账户：babaoshu1.tp

**Comment by yancanlou** [comment]
东东，Famland社区老玩家，3号农场主。
申请多签账号：dvvntlejawcr

**Comment by otto-001** [comment]
空平子
Famland社区玩家，看topal老师文章(UC的救赎系列)参与进来的，原始田投资人。
多签账号：famland33kpz

**Comment by itey** [comment]
xuan
Famland元宇宙白皮书起草人之一，Famand去中心化私域领土元宇宙发起人，Famland社区产品负责人&自愿者，ENS+IPFS dWebsite组织成员，Famland原始田投资人
多签账号：hfdkfamssu5h

**Comment by ECapplebanana** [comment]
Famland社区老玩家。
申请多签账号：sshuiping.tp

**Comment by yuanxingke1** [comment]
大非
26块原始田！
申请多签账号：metaversegod

**Comment by babaoshu9098** [comment]
世界由我造  Famland社区老玩家5 号25号农场主，申请多签账号babaoshu1314

**Reply by yancanlou** [reply]
记错了，25农场主不是你的

**Comment by Bullishftx** [comment]
famland 老玩家
申请多签账号  yle.tag

**Reply by otto-001** [reply]
微信是多少？

**Reply by otto-001** [reply]
需要拉你进微信和电报群，一边及时沟通。方便的话请留言微信和电报号。

**Reply by Bullishftx** [reply]
HY27152493  微信
